README.md

```diff
@@ -155,6 +155,14 @@ This is currently required if any mTLS is used. In future other Spifee implement
 it is possible to use other tools that create the expected certificates for Istio.
 
 ```bash
+# use the default HUB=gcr.io/istio-release, TAG=master-latest-daily defined in global.yaml
+iop istio-system citadel $IBASE/security/citadel
+```
+
+If you want to use a different `HUB` and `TAG`, the following example shows how to install istio 1.3.0's citadel:
+
+```bash
+export HUB=docker.io/istio TAG=1.3.0
```

Comment:
this looks fine - although inconsistent with our developer wiki.  In our developer wiki, we recommend setting HUB and TAG and ISTIO.  See: https://github.com/istio/istio/wiki/Preparing-for-Development#setting-up-environment-variables

I guess I don't know what $IBASE is :)

Reply:
If I set per developer wiki, it won't work in this scenario.  I believe TAG istio is wrong and confusing also.